libs/estdlib/src/gen_server.erl

```diff
@@ -68,29 +68,36 @@
 
 -type init_result(StateType) ::
     {ok, State :: StateType}
-    | {ok, State :: StateType, timeout()}
+    | {ok, State :: StateType, timeout() | {continue, term()}}
     | {stop, Reason :: any()}.
 
+-type handle_continue_result(StateType) ::
+    {noreply, NewState :: StateType}
+    | {noreply, NewState :: StateType, timeout() | {continue, term()}}
+    | {stop, Reason :: term(), NewState :: StateType}.
+
 -type handle_call_result(StateType) ::
     {reply, Reply :: any(), NewState :: StateType}
-    | {reply, Reply :: any(), NewState :: StateType, timeout()}
+    | {reply, Reply :: any(), NewState :: StateType, timeout() | {continue, term()}}
```

Comment:
Just a readability comment here, did you keep `{continue ...}` on the same line of `{reply ...}` for any specific reason?

Reply:
it's "inside" the {reply,..}